packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/InteractionLayer/InteractionLayer.js

```diff
@@ -128,23 +130,25 @@ function InteractionLayer ({
 
   return (
     <g>
-      <DrawingCanvas
-        disabled={disabled || move}
-        pointerEvents={move ? 'none' : 'all'}
-        width={width}
-        height={height}
-        fill='transparent'
-        onPointerDown={onPointerDown}
-        onPointerMove={onPointerMove}
-        onPointerUp={onPointerUp}
-        touch-action='none'
-      />
+      {activeInteractionTask && activeTool && 
+        <DrawingCanvas
+          disabled={disabled || move}
+          pointerEvents={move ? 'none' : 'all'}
+          width={width}
+          height={height}
+          fill='transparent'
+          onPointerDown={onPointerDown}
+          onPointerMove={onPointerMove}
+          onPointerUp={onPointerUp}
+          touch-action='none'
+        />}
       <TranscribedLines
         scale={scale}
       />
       <SubTaskPopup
         onDelete={inactivateMark}
       />
+      <PreviousMarks scale={scale} />
```

Comment:
`PreviousMarks` is trapping pointer events here, preventing the earlier `DrawingCanvas` from receiving them.